[32mINFO    [0m conda_build.metadata:metadata.py:854 Attempting to finalize metadata for conda-build-test-source-setup-py-data-in-env
[32mINFO    [0m conda_build.build:build.py:2280 Packaging conda-build-test-source-setup-py-data-in-env
[32mINFO    [0m conda_build.variants:variants.py:235 Adding in variants from /tmp/tmpa3up8so3/info/recipe/conda_build_config.yaml
[32mINFO    [0m conda_build.utils:utils.py:1970 Renaming work directory '/home/runner/work/_temp/parallel/popen-gw1/test_setup_py_data_in_env0/conda-build-test-source-setup-py-data-in-env_1673038430059/work' to '/home/runner/work/_temp/parallel/popen-gw1/test_setup_py_data_in_env0/conda-build-test-source-setup-py-data-in-env_1673038430059/work_moved_conda-build-test-source-setup-py-data-in-env-1.0-0_linux-64_main_build_loop'
[32mINFO    [0m conda_build.utils:utils.py:1977 shutil.move(work)=/home/runner/work/_temp/parallel/popen-gw1/test_setup_py_data_in_env0/conda-build-test-source-setup-py-data-in-env_1673038430059/work, dest=/home/runner/work/_temp/parallel/popen-gw1/test_setup_py_data_in_env0/conda-build-test-source-setup-py-data-in-env_1673038430059/work_moved_conda-build-test-source-setup-py-data-in-env-1.0-0_linux-64_main_build_loop)